__tests__/app.test.js

```diff
@@ -91,3 +91,27 @@ describe('GET /api/articles/:article_id', () => {
 			});
 	});
 });
+
+describe('GET /api/articles/', () => {
+	test('200: return all articles', () => {
+		return request(app)
+			.get(`/api/articles/`)
+			.expect(200)
+			.then(({ body }) => {
+				const { articles } = body;
+				console.log(typeof articles);
+				for (const article of articles) {
+					expect(article).toMatchObject({
+						article_id: expect.any(Number),
```

Comment:
we also need to explicitly check that the body property is not present in the objects - as we've used toMatchObject - this ignores any properties not specified in the given object. so if the body property is there, it will be ignored and the test will pass (but we want it to fail if the body is present)